scripts/idfFailures.txt

```diff
@@ -0,0 +1 @@
+1) https://github.com/apache/incubator-dubbo,737f7a7ea67832d7f17517326fb2491d0a086dd7,dubbo-registry/dubbo-registry-default|dubbo-dependencies-bom         - this project had a problem with ____
```

Comment:
Is this missing some notes?

Reply:
The following error occurs in this project:

ERROR (WRITE_ERROR_STDOUT_THROWABLE): org.apache.dubbo:dubbo-registry-default:2.7.0-SNAPSHOT
java.lang.Exception: Non-zero exit code (output in /home/aksheeth_i/iDFlakies/scripts/testing-script-results/incubator- dubbo/failing-test-output-1637730318052-a56306f5-cf6b-4e4b-a8db-6bb7817cd974): 1        
         at edu.illinois.cs.testrunner.runner.Runner.$anonfun$runWithCp$3(Runner.scala:71)                                                                                                                      
         at edu.illinois.cs.testrunner.util.TempFiles$.$anonfun$withProperties$2(TempFiles.scala:38)                                                                                                            
         at edu.illinois.cs.testrunner.util.autoClose.$anonfun$autoCloseTry$1(autoClose.scala:7)
        ....

Should this be detailed or removed altogether?

Reply:
Will open issue request and delete idfFailures.txt